assignments/objects.js

```diff
@@ -19,27 +19,82 @@ const example = {
 
 // Write your intern objects here:
 
+const MitziMelloy = {
+  "id" : 1,
+  "name": "Mitzi",
+  "email": "mmelloy@psu.edu",
+  "gender": "F"
+}
+
+const KennanDiben = {
+  "id" : 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const KevenMummery = {
+  "id" : 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const GannieMartinson = {
+  "id" : 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const AntoniettaDaine = {
+  "id" : 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+    
+ console.log(MitziMelloy.name)
 
 // Kennan's ID
 
+ console.log(KennanDiben.id)
+
 // Keven's email
 
+ console.log(KevenMummery.email)
+
 // Gannie's name
 
+ console.log(GannieMartinson.name)
+
 // Antonietta's Gender
 
+ console.log(AntoniettaDaine.gender)
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+let speaker= 'Keenan'
+let speech= 'Hello, my name is ' 
+console.log(speech + speaker)
+
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+  function multiplyNums(num1,num2) {
+  return num1*num2
```

Comment:
For this challenge this function would have to be written to or in the `antonietta` object.